src/eaeInterface.js

```diff
@@ -180,6 +180,7 @@ EaeInterface.prototype._setupInterfaceControllers = function() {
     // Manage the users who have access to the platform - Admin only
     _this.app.post('/user/', _this.usersController.getUser)
         .post('/user/create', _this.usersController.createUser)
+        .post('/user/getAllOf', _this.usersController.getAllUsers)
```

Comment:
getAllOf sounds very weird and doesn't convey what the route actually does.

Can you please rename it to getAllUsers ?

Reply:
done. thought /user/getAll would be better since it s consistent with /job/getAll